var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -106,7 +106,7 @@ class Adios2(CMakePackage, CudaPackage):
     depends_on("c-blosc", when="+blosc")
     depends_on("bzip2", when="+bzip2")
     depends_on("libpng@1.6:", when="+png")
-    depends_on("zfp@0.5.1:", when="+zfp")
+    depends_on("zfp@0.5.1:0.5.99", when="+zfp")
```

Comment:
If you have reason to believe that 0.5.100 or 0.6.0 would work, you should use `@0.5.1:0.5` or `@0.5.1:0`, respectively.

Reply:
Thanks, I'll put
```suggestion
    depends_on("zfp@0.5.1:0.5", when="+zfp")
```
to stay with semantic versioning (which they also express as far as I can see in their CMake config file)